src/features/bookings/components/BookingDetailsTicketContent.native.test.tsx

```diff
@@ -40,21 +40,13 @@ describe('BookingDetailsTicketContent', () => {
         },
       },
     }
-    it('should display the QR code when proDisableEventsQrcode is false', () => {
-      const { getByTestId } = render(
-        <BookingDetailsTicketContent booking={initialBooking} proDisableEventsQrcode={false} />
-      )
-      getByTestId('qr-code')
-    })
 
-    it('should display the QR code when proDisableEventsQrcode is true and event subcategory is not in notQrCodeSubcategories', () => {
-      const { getByTestId } = render(
-        <BookingDetailsTicketContent booking={initialBooking} proDisableEventsQrcode={true} />
-      )
+    it('should display the QR code when event subcategory is not in notQrCodeSubcategories', () => {
```

Comment:
je trouve que `notQrCodeSubcategories` est trop technique dans la règle du test

en plus, ça fait double négation ("**not** in **not**QrCodeSubcategories"), ce qui rend la lecture plus difficile